hustle/main/templates/main/other_profile.html

```diff
@@ -41,9 +85,11 @@ <h1>
             </div>
             <div class="m-4">
                 {% if bl %}
-                <a class="btn btn-secondary" href="{% url 'main:blackListUser' view_user.id %}">Undo BlackList {{ view_user.username }}</a>
+                <a class="btn btn-secondary" href="{% url 'main:blackListUser' view_user.id %}">Undo BlackList {{
+                    view_user.username }}</a>
```

Comment:
Template insertion ends at newline

```suggestion
                <a class="btn btn-secondary" href="{% url 'main:blackListUser' view_user.id %}">
                    Undo BlackList {{ view_user.username }}
                </a>
```